libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/EntityAssignmentComparer.cs

```diff
@@ -31,6 +31,12 @@ public EntityAssignmentComparer(string[] operations)
             operationPreference = operations;
         }
 
+        /// <summary>
+        /// Compares <see cref="EntityAssignment"/> x against y to determine its relative priority.
```

Comment:
This sounds weird to me, maybe
```suggestion
        /// Compares two <see cref="EntityAssignment"/> to determine its relative priority.
```
?

Reply:
I stand by the original proposed change, I find it more helpful since it clarifies that the result is relative to x, not y.